doc/release/1.15.0-notes.rst

```diff
@@ -127,6 +131,7 @@ will be removed in the future.
 
 The ``NpzFile`` returned by ``np.savez`` is now a ``collections.abc.Mapping``
 -----------------------------------------------------------------------------
+---------------------------------------------------------------------------
```

Comment:
This fix is incorrect.

Reply:
fixed

Reply:
Fixed but it is still marked as changed on github. I wonder why